test/to_do_app/todos_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule ToDoApp.TodosTest do
+  use ToDoApp.DataCase
+
+  alias ToDoApp.Todos
+
+  describe "items" do
+    alias ToDoApp.Todos.Items
```

Comment:
Keep the on top and not inside the `describe` macro. 

https://github.com/christopheradams/elixir_style_guide?tab=readme-ov-file#module-attribute-ordering

Refer this , there is a conventional order in which `alias` and similar statements should be stacked up on the top. Refactor the code.